themes/bootstrap3/templates/search/results-list.phtml

```diff
@@ -0,0 +1,9 @@
+<?php
+  // Create a css class for results from the search class:
+  $searchClass = $searchClassCss = $this->params->getSearchClassId();
+  if (!ctype_upper($searchClassCss)) {
```

Comment:
It took me a minute to figure out why this was here. While we're touching the code anyway, maybe it's worth adding a comment like:

`// Convert camelCase to hyphenated-text, unless it's all-uppercase, like EDS`

Reply:
Good idea, comment added.